fe/fe-core/src/main/java/org/apache/doris/statistics/ColStatsData.java

```diff
@@ -54,12 +54,12 @@ public class ColStatsData {
 
     public ColStatsData(ResultRow row) {
         this.statsId = new StatsId(row);
-        this.count = Long.parseLong(row.get(7));
-        this.ndv = Long.parseLong(row.getWithDefault(8, "0"));
-        this.nullCount = Long.parseLong(row.getWithDefault(9, "0"));
+        this.count = (long) Double.parseDouble(row.get(7));
```

Comment:
why convert count from Double?

Reply:
Since ratio is a float so result also would be float type